test/controllers/actions/FakeCheckOtherCountryRegistrationFilter.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.actions
+
+import models.requests.AuthenticatedIdentifierRequest
+import play.api.mvc.Result
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class FakeCheckOtherCountryRegistrationFilter extends CheckOtherCountryRegistrationFilter {
+
+  override protected def filter[A](request: AuthenticatedIdentifierRequest[A]): Future[Option[Result]] = {
+    Future.successful(None)
+  }
+
+  override protected def executionContext: ExecutionContext =
+    scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Could we add a the implicit value as with other class' rather than importing the global EC?